Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -149,6 +149,18 @@ theorem measure_union_add_inter' (hs : MeasurableSet s) (t : Set α) :
   rw [union_comm, inter_comm, measure_union_add_inter t hs, add_comm]
 #align measure_theory.measure_union_add_inter' MeasureTheory.measure_union_add_inter'
 
+lemma measure_symmDiff_eq (hs : MeasurableSet s) (ht : MeasurableSet t) :
+    μ (s ∆ t) = μ (s \ t) + μ (t \ s) := by
+  simpa only [symmDiff_def, sup_eq_union] using measure_union disjoint_sdiff_sdiff (ht.diff hs)
+
+lemma measure_symmDiff_le (s t u : Set α) :
+    μ (s ∆ u) ≤ μ (s ∆ t) + μ (t ∆ u) := by
+  suffices : s ∆ u ⊆ (s ∆ t) ∪ (t ∆ u)
```

Comment:
This one would deserve to be a lemma of its own.

Reply:
Not only does it deserve to be a lemma of its own, but it already exists! I have golfed the proof accordingly.